functions-python/update_validation_report/src/main.py

```diff
@@ -0,0 +1,187 @@
+#
+#   MobilityData 2024
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+import logging
+import os
+from time import sleep
+from typing import List
+
+import functions_framework
+import requests
+import sqlalchemy.orm
+from google.cloud import storage
+from sqlalchemy.engine import Row
+from sqlalchemy.engine.interfaces import Any
+
+from database_gen.sqlacodegen_models import Gtfsdataset, Gtfsfeed
+from helpers.database import start_db_session
+from helpers.logger import Logger
+
+logging.basicConfig(level=logging.INFO)
+
+
+@functions_framework.http
```

Comment:
I can't see who calls this endpoint.

Reply:
no process currently calls this endpoint -- it has to be manually called or part of the release process of the GTFS validator after the web validator has been updated